core/src/main/resources/jenkins/views/JenkinsHeader/headerContent.jelly

```diff
@@ -1,6 +1,10 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:h="/lib/layout/header">
-    <header id="page-header" class="page-header">
+    <header id="page-header"
+            class="page-header"
+            data-root-url="${rootURL}"
+            data-search-url="${rootURL + '/search/suggest'}"
+            data-search-help-url="${searchHelpUrl}">
```

Comment:
This location looks like it would break with https://plugins.jenkins.io/customizable-header/ and other plugins implementing custom headers.

Reply:
fixed in https://github.com/jenkinsci/jenkins/pull/7569/commits/6092031216b01211ca612a143ca826d709337739